src/main/scala/pipedsl/Parser.scala

```diff
@@ -151,11 +151,11 @@ class Parser(rflockImpl: String) extends RegexParsers with PackratParsers {
   }
 
   lazy val methodCall: P[ECall] = positioned {
-    iden ~ "." ~ iden ~ parens(repsep(expr, ",")) ^^ { case i ~ _  ~ n ~ args => ECall(i, Some(n), args) }
+    iden ~ "." ~ iden ~ parens(repsep(expr, ",")) ^^ { case i ~ _  ~ n ~ args => ECall(i, Some(n), args, false) }
   }
 
   lazy val simpleAtom: P[Expr] = positioned {
-    "call" ~> iden ~ parens(repsep(expr, ",")) ^^ { case i ~ args => ECall(i, None, args) } |
+    "call" ~> angular("atomic" | "a").? ~ iden ~ parens(repsep(expr, ",")) ^^ { case a ~ i ~ args => ECall(i, None, args, a.isDefined) } |
```

Comment:
Somehow I think `call<a> subpipe(args)` makes more sense, because atomic call to pipeline isn't a property of the pipeline, unlike memory. This is more like doing one specific pipeline call atomically.